Modules/Models/Configuration.swift

```diff
@@ -16,396 +16,401 @@ import ExposureNotification
+
+    /// The url of the privacy notice
+    /// - note: this dictionary uses string as a key because only strings and ints
+    /// are really considered as dictionaries by Codable
+    /// https://bugs.swift.org/browse/SR-7788
+    public let privacyNoticeURL: [String: URL]
+
+    /// The url of the terms of use
+    /// - note: this dictionary uses string as a key because only strings and ints
+    /// are really considered as dictionaries by Codable
+    /// https://bugs.swift.org/browse/SR-7788
+    public let termsOfUseURL: [String: URL]
+
+    /// The urls of the FAQs for the various languages
+    /// - note: this dictionary uses string as a key because only strings and ints
+    /// are really considered as dictionaries by Codable
+    /// https://bugs.swift.org/browse/SR-7788
+    public let faqURL: [String: URL]
+
+    /// Probability with which the app sends analytics data in case of match. Value in the [0, 1] range.
+    public let operationalInfoWithExposureSamplingRate: Double
+
+    /// Probability with which the app sends analytics data in case of non match. Value in the [0, 1] range.
+    public let operationalInfoWithoutExposureSamplingRate: Double
 
-    /// Scores that indicate Bluetooth signal strength.
-    public let attenuationBucketScores: [UInt8]
+    /// Mean of the exponential distribution that regulates the execution of dummy analytics requests
+    public let dummyAnalyticsMeanStochasticDelay: Double
 
-    /// The weight applied to a Bluetooth signal strength score.
-    public let attenuationWeight: Double
+    /// Average wait time (in seconds) from one dummy ingestion request to the next one.
+    public let dummyIngestionAverageRequestWaitingTime: Double
 
-    /// Scores that indicate the days since the user’s last exposure.
-    public let daysSinceLastExposureBucketScores: [UInt8]
+    /// Arrays of probabilities that describes the chance the app has after each dummy ingestion request to send another one.
+    /// The i-th element represents the chance of sending a request after the i-th request.
+    public let dummyIngestionRequestProbabilities: [Double]
 
-    /// The weight assigned to a score applied to the days since the user’s exposure.
-    public let daysSinceLastExposureWeight: Double
+    /// Mean of the exponential distribution that regulates the execution of dummy ingestion requests
+    public let dummyIngestionMeanStochasticDelay: Double
 
-    /// Scores that indicate the duration of a user’s exposure.
-    public let durationBucketScores: [UInt8]
+    /// Duration of the window of opportunity the app has to send a dummy ingestion request.
+    public let dummyIngestionWindowDuration: Double
 
-    /// The weight assigned to a score applied to the duration of the user’s exposure.
-    public let durationWeight: Double
+    /// Average wait time (in seconds) from the start of a foreground session before starting a simulated dummy ingestion sequence.
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: Modules/Models/Configuration.swift;
  Line: 131;
-->

- :warning: swiftlint (0.39.2): Line should be 130 characters or less: currently 131 characters (line_length)